### ISSUE TITLE
Suggest：Registry operation analysis

### ISSUE THREAD

**Issue by mushixun** [vmware/harbor#5519]
Registry operation analysis
     1. Provide the statistics of the total number of pull, push, and delete in a specified time period
     2. Provide top ranking
           Number of image pulls top ranking 
           Number of image tags top ranking
           Image size top ranking (mirror maximum and minimum dimension statistics)
           Image update frequency top ranking (statistics by specified time period)
     3. Tenant analysis
           Provide tenants with data analysis of three dimensions through pull, push, and delete operations in a specified time period, and do top ranking
     4. Image library analysis
          Provides the total number of projects at the specified time, the total number of mirrors, the total number of users, and provides trend analysis capabilities (default 12 hours), and can view historical performance data trend graphs for up to 2 months
     5. Change analysis
         The specified time period, number of images, number of items, number of users, pull, push, delete number and percentage change of the ring

This is our operational analysis function based on harbor development. If it is useful, hope submit it to the community

**Comment by reasonerjt** [comment]
@mushixun that would be very cool!  Would you join the community meeting and share the idea?  Then we may explore how it can be integrated?

**Comment by mushixun** [comment]
Ok,what need i to do？

**Comment by steven-zou** [comment]
@mushixun 

Could you please subscribe the harbor-dev mail list first? 
```
Developer Group: Join Harbor developer group: harbor-dev@googlegroups.com for discussion on Harbor development and contribution. To subscribe, send an email to harbor-dev+subscribe@googlegroups.com.
```

The community conf call invitation will be sent out by that mail.

I'll schedule an ad-hoc community conf call for you to give us an introduction about what you and your team have done and notify u with this issue thread. (For easy communication, if possible, can u share me your mail?)

Regarding the conf call, can you tell me what's your time zone and potentially available time slots? Currently, our community call is always scheduled for at 6:00-7:00 AM.

**Comment by clouderati** [comment]
I agree – would absolutely love to hear about this during our community call. Looking forward to the discussion @mushixun!

**Comment by mushixun** [comment]
Understand，I will join your community call。

**Comment by steven-zou** [comment]
@mushixun 

Any way to easily get in touch with u? mail or something else?

**Comment by mushixun** [comment]
Email:10288345@qq.com
QQ:10288345

**Comment by steven-zou** [comment]
@mushixun 

I have put your feature topic into the community call queue, please check the wiki page: https://github.com/goharbor/harbor/wiki/Community-Conference-Calls.

**Comment by steven-zou** [comment]
@mushixun 

 We're planning to use md file as a proposal content format and use PR raised in [goharbor/harbor](https://github.com/goharbor/community) for reviewing.

  Would u please follow the guideline below to move your proposal to there?

  ```
  git clone https://github.com/goharbor/community
  git checkout -b YOUR_BRANCH
  # add md file with meaningful-words_proposal.md name under `proposals/new` folder
  git commit
  # raise PR to request merge proposal
  # reviewers can append comments to the PR
  ```

**Comment by steven-zou** [comment]
Assign @mushixun as the owner of this proposal.

**Comment by mushixun** [comment]
Ok,I will commit later,Thanks! @steven-zou
